File: backend-ruby/modules/catalog/infrastructure/seed_data.rb
module Catalog
  module Infrastructure
    SEED_PRODUCTS = [
      { sku: "SKU-1001", name: "Wireless Mechanical Keyboard", description: "Hot-swappable switches, USB-C.", price_cents: 8999, category: "electronics", stock_quantity: 42 },
      { sku: "SKU-1002", name: "27in 4K Monitor", description: "IPS panel, 144Hz.", price_cents: 39999, category: "electronics", stock_quantity: 15 },
      { sku: "SKU-1003", name: "Ceramic Pour-Over Kettle", description: "1.2L, gooseneck spout.", price_cents: 4599, category: "home", stock_quantity: 88 },
      { sku: "SKU-1004", name: "Trail Running Shoes", description: "Lightweight, grippy outsole.", price_cents: 12999, category: "apparel", stock_quantity: 60 },
      { sku: "SKU-1005", name: "Stainless Steel Water Bottle", description: "32oz, insulated.", price_cents: 2999, category: "home", stock_quantity: 120 },
    ].freeze
  end
end

File: backend-kotlin/src/main/kotlin/com/example/modules/catalog/infrastructure/SeedData.kt
package com.example.modules.catalog.infrastructure

import kotlinx.serialization.Serializable

@Serializable
data class Product(val sku: String, val name: String, val description: String,
                    val priceCents: Long, val category: String, val stockQuantity: Int)

fun seedProducts() = listOf(
    Product("SKU-1001", "Wireless Mechanical Keyboard", "Hot-swappable switches, USB-C.", 8999, "electronics", 42),
    Product("SKU-1002", "27in 4K Monitor", "IPS panel, 144Hz.", 39999, "electronics", 15),
    Product("SKU-1003", "Ceramic Pour-Over Kettle", "1.2L, gooseneck spout.", 4599, "home", 88),
    Product("SKU-1004", "Trail Running Shoes", "Lightweight, grippy outsole.", 12999, "apparel", 60),
    Product("SKU-1005", "Stainless Steel Water Bottle", "32oz, insulated.", 2999, "home", 120),
)
